packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -738,17 +738,32 @@ const replaceFileLinks = async ({
     return nodeString
   }
 
-  const hrefMatches = execall(
-    /(\\"|\\'|\()([^'"()]*)(\/wp-content\/uploads\/[^'">()]+)(\\"|\\'|>|\))/gm,
-    nodeString
-  )
+  const hrefMatches = [
```

Comment:
can you add an example comment on what you're trying to match? That helps people to understand the regexes better.

Reply:
We could also add a unit test?

Reply:
That's even better

Reply:
Oh good call, there's already one there so I'll expand it a bit

Reply:
Turns out there wasn't one. I ended up adding an integration test instead as well as some comments explaining what each regex is matching for